r-package/grf/R/survival_forest.R

```diff
@@ -0,0 +1,203 @@
+#' Survival forest
+#'
+#' Trains a forest for right-censored surival data that can be used to estimate the
+#' conditional surival function S(t, x) = P[Y > t | X = x]
+#'
+#' @param X The covariates.
+#' @param Y The response time (may be negative).
+#' @param delta The censoring indicator (1: failure, 0: censored)
+#' @param num.trees Number of trees grown in the forest. Default is 1000.
+#' @param sample.weights (experimental) Weights given to an observation in prediction.
```

Comment:
I think we can omit the `sample.weights` parameter for now.

Reply:
It says it is used for predictions only, should this be more emphasized?